spec/motion-support/core_ext/time/conversion_spec.rb

```diff
@@ -1,50 +1,49 @@
-describe "Time" do
```

Comment:
These " -> ' changes seem unneeded, unrelated to the commit, and also break the convention.